testing/unit/tp/account/test_transfer.py

```diff
@@ -0,0 +1,327 @@
+        nonce=time.time().hex().encode(),
+    )
+
+    serialized_header = transaction_header.SerializeToString()
+
+    transaction_request = TpProcessRequest(
+        header=transaction_header,
+        payload=serialized_transaction_payload,
+        signature=create_signer(private_key=ACCOUNT_FROM_PRIVATE_KEY).sign(serialized_header),
+    )
+    mock_context = create_context(account_from_balance=ACCOUNT_FROM_BALANCE, account_to_balance=ACCOUNT_TO_BALANCE)
+
+    with pytest.raises(InvalidTransaction) as error:
+        AccountHandler().apply(transaction_request=transaction_request, context=mock_context)
+
+    assert f'Invalid account method value ({account_method_impossible_value}) has been set.' == str(error.value)
+
+
+def test_account_handler_apply_decode_error():
+    """
+    Case: send transaction request, to send tokens to address, to account handler with invalid transfer method value.
+    Expect: invalid transaction error is raised with invalid account method value error message.
+    """
+    serialized_not_valid_transaction_payload = b'F1120071db7c02f5731d06df194dc95465e9b27'
+
+    transaction_header = TransactionHeader(
+        signer_public_key=RANDOM_NODE_PUBLIC_KEY,
+        family_name=TRANSACTION_REQUEST_ACCOUNT_HANDLER_PARAMS.get('family_name'),
+        family_version=TRANSACTION_REQUEST_ACCOUNT_HANDLER_PARAMS.get('family_version'),
+        inputs=INPUTS,
+        outputs=OUTPUTS,
+        dependencies=[],
+        payload_sha512=hash512(data=serialized_not_valid_transaction_payload),
+        batcher_public_key=RANDOM_NODE_PUBLIC_KEY,
+        nonce=time.time().hex().encode(),
+    )
+
+    serialized_header = transaction_header.SerializeToString()
+
+    transaction_request = TpProcessRequest(
+        header=transaction_header,
+        payload=serialized_not_valid_transaction_payload,
+        signature=create_signer(private_key=ACCOUNT_FROM_PRIVATE_KEY).sign(serialized_header),
+    )
+    mock_context = create_context(account_from_balance=ACCOUNT_FROM_BALANCE, account_to_balance=ACCOUNT_TO_BALANCE)
+
+    with pytest.raises(InvalidTransaction) as error:
+        AccountHandler().apply(transaction_request=transaction_request, context=mock_context)
+
+    assert 'Cannot decode transaction payload.' == str(error.value)
+
+
+def test_account_transfer_from_address():
+    """
+    Case: transfer tokens from address to address.
+    Expect: account's balances, stored in state, are changed according to transfer amount.
+    """
+    expected_account_from_balance = ACCOUNT_FROM_BALANCE - TOKENS_AMOUNT_TO_SEND
+    expected_account_to_balance = ACCOUNT_TO_BALANCE + TOKENS_AMOUNT_TO_SEND
+
+    transfer_payload = TransferPayload()
+    transfer_payload.address_to = ACCOUNT_ADDRESS_TO
+    transfer_payload.value = TOKENS_AMOUNT_TO_SEND
+
+    mock_context = create_context(account_from_balance=ACCOUNT_FROM_BALANCE, account_to_balance=ACCOUNT_TO_BALANCE)
+
+    result = AccountHandler()._transfer_from_address(
```

Comment:
@eugene-babichenko, do you think, if class has 1000 code line, but only 5 of them are public, we should not to test 995 code line? :)

Also actually those methods aren't private because public key class and atomic swap class use them.